docs/examples/workflows/callable_script.md

```diff
@@ -71,11 +77,28 @@
         return Output(output=[Input.parse_raw(input)])
 
 
+    # Use the script_annotations feature to seamlessly enable aliased kebab-case names
+    # as your template interface, while using regular snake_case in the Python code
+    @script()
+    def function_kebab(
+        a_but_kebab: Annotated[int, Parameter(name="a-but-kebab")] = 2,
+        b_but_kebab: Annotated[str, Parameter(name="b-but-kebab")] = "foo",
+    ) -> Output:
+        return Output(output=[Input(a=a_but_kebab, b=b_but_kebab)])
+
+
+    @script()
+    def function_kebab_object(input_values: Annotated[Input, Parameter(name="input-values")]) -> Output:
+        return Output(output=[input_values])
+
+
     with Workflow(name="my-workflow") as w:
         with Steps(name="my-steps") as s:
             my_function(arguments={"input": Input(a=2, b="bar")})
             str_function(arguments={"input": Input(a=2, b="bar").json()})
             another_function(arguments={"inputs": [Input(a=2, b="bar"), Input(a=2, b="bar")]})
+            function_kebab(arguments={"a_but_kebab": 3, "b_but_kebab": "bar"})
```

Comment:
Are we sure Argo lint will let us do this? At a yaml level the mapping has a discrepancy. Let's confirm.

Reply:
I think this was an accident, should be `a-but-kebab` instead

Reply:
It does actually work though, interestingly... not sure about naming limits on parameters. This is accepted and runs:

```yaml
apiVersion: argoproj.io/v1alpha1
kind: Workflow
metadata:
  generateName: my-workflow-
spec:
  entrypoint: whalesay
  templates:
    - name: whalesay
      container:
        args:
          - '{{inputs.parameters.test_parameter}}'
        command:
          - cowsay
        image: docker/whalesay:latest
      inputs:
        parameters:
          - name: test_parameter
            value: The name has an underscore
```